Removing rhscon and create new dashboard vm

jtwill98 opened the pull request.

gaelrehault commented:
src/pilot/config_dashboard.py still has a bunch of rhscon references, assuming this will be addressed later ? 

Also references in : 
src/update_upgrade/ceph_upgrade_director.sh
src/update_upgrade/ceph_upgrade_sah.sh
src/update_upgrade/upgrade-js6.0.1.sh

We need to talk about how we want to handle those with @dpaterson

heydog15 commented:
In src/pilot/config_dashboard.py, the references to the BZ needs to be taken out.  Won't apply to Dashboard and that issue was fixed in ceph-ansible a while back.

Also in deploy-dashboard-vm.py, the Copyright needs to have 2018 added. Shouldn't the  dashboard-post.log be copied into the dashboard-ks-logs directory so all logs are together?

jtwill98 commented:
I addressed some of the concerns.   Yes, I did not forget the update_upgrade scripts.  I will address these with Dave after fixing the config_dashboard.   What is here is the VM deployment and calls to it from automation.   The next step will be to finalize the changes config_dashboard script.  What's there current, is only the user and etc/hosts prep.  Running ansible to install the dashboard and collectors is part of the changes to finalize.

dpaterson commented on src/pilot/customize_image.sh:
```diff
@@ -53,13 +53,12 @@ fi
 )
 
 packages=(
     calamari-server
```
We still need calamari server for cephmetrics?

jtwill98 replied:
That has yet to be determined.  I left it in during my investigation.  I will verify if it's needed during the integration effort.

pauldardeau commented on src/deploy/osp_deployer/checkpoints.py:
```diff
@@ -288,9 +288,9 @@ def rhscon_vm_health_check(self):
 
         logger.debug("*** Verify the Storage Console VM can ping "
```
Nothing urgent, but we might need to change the name here away from Storage Console

jtwill98 replied:
NP.

pauldardeau commented on src/deploy/osp_deployer/checkpoints.py:
```diff
@@ -300,9 +300,9 @@ def rhscon_vm_health_check(self):
         logger.debug(
             "*** Verify the Storage Console VM can ping the SAH node "
```
Same here

pauldardeau commented on src/mgmt/dashboard.cfg:
```diff
@@ -1,4 +1,4 @@
-# (c) 2014-2018 Dell
```
Remove copyright indicator?

cdearborn commented:
+1 Looks good.
